CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.17.0 (2019-07-11)
+
+- Added a `getUrl()` method to `BBAuth` that will translate a 1BB tokenized URL and zone to a zoned URL. For example, a tokenized URL of `1bb://eng-hub00/version` and a zone of `eng-pusa01` would translate to `https://eng-pusa01.app.blackbaud.net/hub00/version`. [#90](https://github.com/blackbaud/auth-client/pull/90)
```

Comment:
Not a big deal, but I'd change "that will translate" to "that translates" and "would translate" to just "translate"